### PR TITLE
ocrvs-3636:  Empty state text

### PR DESCRIPTION
**Pull request by CornelMaze**
- Added Empty state to the vsexport page.

### CODE REVIEW

File: packages/client/src/views/SysAdmin/Performance/Vsexports/VSExport.tsx [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+import { LoadingIndicator } from '@client/views/OfficeHome/LoadingIndicator'
+
+import { MINIO_URL } from '@client/utils/constants'
+import { DynamicHeightLinkButton } from '@client/views/Settings/items/components'
+import { ErrorToastNotification } from '@opencrvs/components/lib/interface'
+
+import { buttonMessages, errorMessages } from '@client/i18n/messages'
+import { Event, GetVsExportsQuery, VsExport } from '@client/utils/gateway'
+
+const UserTable = styled(BodyContent)`
```
Comment (jpye-finch) [inline packages/client/src/views/SysAdmin/Performance/Vsexports/VSExport.tsx:39]: @CornelMaze 
Why is this needed? Is it because we arn't using < Frame / >

Comment (CornelMaze) [reply packages/client/src/views/SysAdmin/Performance/Vsexports/VSExport.tsx:39]: I'll check it out.

File: packages/components/src/components/icons/Export.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+import React from 'react'
+import { colors } from '../colors'
+
+export function Export({
```
Comment (jpye-finch) [inline packages/components/src/components/icons/Export.tsx:15]: This file is not required. We can use < Icon />. It is a feather icon called 'upload'

Comment (CornelMaze) [reply packages/components/src/components/icons/Export.tsx:15]: ok

File: packages/client/src/views/SysAdmin/Performance/Vsexports/VSExport.tsx [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+          fileSize: item.fileSize
+        })
+
+        return (
+          <ListViewSimplified key={`${item.createdOn}_${item.event}`}>
+            <ListViewItemSimplified
+              label={<Label id={`${item.year}_label`}>{item.year}</Label>}
+              value={<Value id={`${item.createdOn}_value`}>{sizeValue}</Value>}
+              actions={
+                <DynamicHeightLinkButton
```
Comment (jpye-finch) [inline packages/client/src/views/SysAdmin/Performance/Vsexports/VSExport.tsx:86]: where is this defined?
Can we not just use < Link /> from the component library?

File: packages/client/src/views/SysAdmin/Performance/Vsexports/VSExport.tsx [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+            }
+          >
+            <Query<GetVsExportsQuery>
+              query={GET_TOTAL_VSEXPORT}
+              fetchPolicy={'no-cache'}
+            >
+              {({ data, loading, error }) => {
+                if (error) {
+                  return (
+                    <ErrorToastNotification
```
Comment (jpye-finch) [inline packages/client/src/views/SysAdmin/Performance/Vsexports/VSExport.tsx:148]: Let's use <Toast /> (type error) here from the component library. ErrorToastNotification is depreciated

File: packages/client/src/views/SysAdmin/Performance/Vsexports/VSExport.tsx (hunk 1 above)
Comment (CornelMaze) [reply packages/client/src/views/SysAdmin/Performance/Vsexports/VSExport.tsx:39]: @jpye-finch Yes. Is the Frame available?

Comment (jpye-finch) [reply packages/client/src/views/SysAdmin/Performance/Vsexports/VSExport.tsx:39]: yes
https://ui-kit.farajaland-qa.opencrvs.org/?path=/docs/layout-frame--page-template-content-large
